data/nitin-787.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Nitin Sharma",
+  "bio": "Playing with linux **Flutter dev**. Travel enthusiast & Cosmic Historian Extraordinaire"
+  "links": [
+    {
+      "name": "wanna collab on github"
```

```suggestion
      "name": "wanna collab on github",
```
missing commas in  all fields below